en/docs/Administer/ProductSecurity/enabling-java-security-manager.md

```diff
@@ -1,157 +1,159 @@
-# admin\_Enabling Java Security Manager
+# Enabling Java Security Manager
 
 !!! warning
-**Please note that this document is currently under review. Currently, WSO2 does not recommend Java security manager for production environments due to some known issues.**
+    **Please note that this document is currently under review. Currently, WSO2 does not recommend Java security manager for production environments due to some known issues.**
 
 
 The Java Security Manager is used to define various security policies that prevent untrusted code from manipulating your system. Enabling the Java Security Manager for WSO2 products activates the Java permissions that are in the `<PRODUCT_HOME>/repository/conf/sec.policy` file. You modify this file to change the Java security permissions as required.
 
 !!! info
-**Before you begin**
+    **Before you begin**
 
--   Ensure that you have Java 1.8 installed.
--   Note that you need to use a keystore for signing JARs using the Java security manager. In this example, you will be using the default keystore in your WSO2 product ( `wso2carbon.jks` ). You can read about the recommendations for using keystores from [here](https://docs.wso2.com/display/ADMIN44x/Using+Asymmetric+Encryption#UsingAsymmetricEncryption-RecommendationsforsettingupkeystoresinWSO2products) .
+    * Ensure that you have Java 1.8 or Java 11 installed.
+    * Note that you need to use a keystore for signing JARs using the Java security manager. In this example, you will be using the default keystore in your WSO2 product ( `wso2carbon.jks` ). You can read about the recommendations for using keystores from [here](https://docs.wso2.com/display/ADMIN44x/Using+Asymmetric+Encryption#UsingAsymmetricEncryption-RecommendationsforsettingupkeystoresinWSO2products).
```

Comment:
We should not point to the Admin guide. This content should be already available in the APIM docs

Reply:
Fixed in f906106